spring-cloud-gcp-starters/spring-cloud-gcp-starter-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/GcpPubSubProperties.java

```diff
@@ -59,5 +59,4 @@ public void setProjectId(String projectId) {
 	public Credentials getCredentials() {
 		return this.credentials;
 	}
-
```

Comment:
Blank like before the end of class.
Well, actually it sounds better like: blank like after each method and property. Or if you want: surround class members with blank lines.

Reply:
Sounds like something that can be enforced with checkstyle.

Reply:
That's good idea, but I'm not aware of such a checker there: http://checkstyle.sourceforge.net/checks.html. As well as there is no the way to validate JavaDoc style we have talked before...

Reply:
Maybe this one? http://checkstyle.sourceforge.net/config_whitespace.html#EmptyLineSeparator

Reply:
Oh! Great! Let's give it a shot! 😄

Reply:
We can! :)
However, looking back at the [Spring code style guide](https://github.com/spring-projects/spring-framework/wiki/Code-Style#blank-lines), I'm not seeing the requirement to put blank lines after methods. Where are you getting this style requirement from?

Reply:
We follow it in our projects. That's really so simple: just surround class members with blank lines.
The SF Docs just might be a bit out dated.

Do you really always need some official doc for confirmation though?..

Reply:
An official doc is handy and automatic enforcement is even better!
That is if you care about these rules and want them followed when many developers are involved and new ones get on-boarded.